src/Application/User/Command/SaveReportAddressCommandHandler.php

```diff
@@ -29,5 +38,103 @@ public function __invoke(SaveReportAddressCommand $command): void
         $reportAddress->setCreatedAt($this->dateUtils->getNow());
 
         $this->reportAddressRepository->add($reportAddress);
+
+        // Send report to IGN API
+        $this->sendReportToIgn($command);
+    }
+
+    private function sendReportToIgn(SaveReportAddressCommand $command): void
+    {
+        $comment = $command->content;
+        $userId = $command->user->getUuid();
+
+        // Try to get geometry from roadBanId first (for entire road), fallback to organization geometry
+        $roadBanIdGeometry = $command->roadBanId ? $this->getGeometryFromRoadBanId($command->roadBanId) : null;
+        $geometry = ($roadBanIdGeometry && trim($roadBanIdGeometry) !== '')
```

Comment:
```suggestion
        $geometry = $roadBanIdGeometry ?? $this->getOrganizationPointGeometry($command);
```

Je crois que tu peux faire comme ça